src/qt/pivx/coldstakingwidget.cpp

```diff
@@ -287,9 +287,14 @@ void ColdStakingWidget::onContactsClicked(){
         menu->hide();
     }
 
-    int contactsSize = isContactOwnerSelected ? walletModel->getAddressTableModel()->sizeSend() : walletModel->getAddressTableModel()->sizeColdSend();
+    int contactsSize = isContactOwnerSelected ? walletModel->getAddressTableModel()->sizeRecv() : walletModel->getAddressTableModel()->sizeColdSend();
     if(contactsSize == 0) {
-        inform(tr("No contacts available, you can go to the contacts screen and add some there!"));
+        inform(tr(
+                (isContactOwnerSelected ?
+                "No receive addresses available, you can go to the receive screen and create some there!" :
+                "No contacts available, you can go to the contacts screen and add some there!"
+                )
+        ));
```

Comment:
This won't be properly picked up for localization in Transifex. Suggest the below instead:
```cpp
inform((isContactOwnerSelected ?
    tr("No receive addresses available, you can go to the receive screen and create some there!") :
    tr("No contacts available, you can go to the contacts screen and add some there!")
    )
);
```

Reply:
done, updated.